dist/tools/sliptty/start_network.sh

```diff
@@ -89,8 +101,8 @@ start_uhcpd() {
 }
 
 usage() {
-    echo "usage: $1 [-I <sl0>] <prefix> serial [baudrate]"
-    echo "usage: $1 [-I <sl0>] <prefix> tcp:host [port]"
+    echo "usage: $1 [-I <sl0>] [-e] [-g <addr>/<prefix_len>] <prefix> serial [baudrate]"
+    echo "usage: $1 [-I <sl0>] [-e] [-g <addr>/<prefix_len>] <prefix> tcp:host [port]"
```

Comment:
`examples/gnrc_border_router/Makefile.slip.conf` could make use of the `-e` option.